libs/mel/src/lib/pagination-buttons/pagination-buttons.component.html

```diff
@@ -0,0 +1,44 @@
+<div class="relative">
+  <div class="flex flex-row items-center">
+    <gn-ui-button
+      type="outline"
+      [disabled]="currentPage === 1"
+      (buttonClick)="previousPage()"
+      extraClass="pagination-btn-nav mr-14"
+    >
+      <mat-icon class="pagination-btn-arrow material-symbols-outlined"
+        >chevron_left</mat-icon
+      >
+      <span class="font-bold ml-1" translate
+        >mel.datahub.search.pagination.previous</span
+      >
+    </gn-ui-button>
+    <ng-container *ngFor="let page of visiblePages">
```

Comment:
I think here you can use the new syntax with `@for`and `@if`.

Reply:
Indeed, thanks!